src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -202,6 +202,18 @@ public async Task DispatchEvent(string eventDescriptorJson, string eventArgs)
             }
         }
 
+        internal void OnRenderCompleted(long renderId, string errorMessageOrNull)
+        {
+            try
+            {
+                Renderer.OnRenderCompleted(renderId, errorMessageOrNull);
+            }
+            catch (Exception e)
```

Comment:
It looks like the change here is to stop calling `HandleException` from `RemoteRenderer`'s `OnRenderCompleted`, and instead have that throw, but catch the exception here and then pass it on to the equivalent `UnhandledException`.

My awareness of all the code paths isn't super fresh, but it seems more or less the same result. Could you clarify for me why we're making this change? What different scenarios does it handle?

Reply:
It does less logging.